htdocs/web_portal/views/downtime/edit_downtime.php

```diff
@@ -225,8 +225,10 @@ class="form-control" id="Select_Sites" name="select_sites" size="10"
         $('#startDate').data("DateTimePicker").date("<?php echo date_format($startDate,"d/m/Y"); ?>");
         $('#endDate').data("DateTimePicker").date("<?php echo date_format($endDate,"d/m/Y"); ?>");
 
-        //Set the start and finish times (don't echo in the full date, just the time values, time widget didn�t like timestamp with date)
-        $('#startTime').data("DateTimePicker").date("<?php echo date_format($startDate,"H:i"); ?>");
+        /**
+         * Set the start and finish times (don't echo in the full date,
+         * just the time values, time widget didn't like timestamp with date
```

Comment:
```suggestion
         * just the time values, time widget didn't like timestamp with date)
```
the bracket is never closed :)

Reply:
Ah right!. 👍